modules/ROOT/pages/csharp.adoc

```diff
@@ -876,7 +884,7 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
 
 === 2.0.2
 
-* Support for Log redaction of sensitive information when <<logging,logging is enabled>>.
+include::{partialsdir}/feature-LogRedaction-2.0.2.adoc[]
```

Comment:
ah. I see what you mean now. Just copying what I'd seen. I assumed that {partialsdir} was defined somewhere in case antora changed the way it worked.  Np, I'll use this syntax from now on.